addons/account/models/account_journal_dashboard.py

```diff
@@ -215,8 +214,12 @@ def get_journal_dashboard_datas(self):
             (number_waiting, sum_waiting) = self._count_results_and_sum_amounts(query_results_to_pay, currency, curr_cache=curr_cache)
             (number_draft, sum_draft) = self._count_results_and_sum_amounts(query_results_drafts, currency, curr_cache=curr_cache)
             (number_late, sum_late) = self._count_results_and_sum_amounts(late_query_results, currency, curr_cache=curr_cache)
+            lines_to_check_ids = self.env['account.move.line'].search([('journal_id', '=', self.id), ('move_id.to_check', '=', True)])
```

Comment:
Can we add new stored related field (to_check) in account.move.line (move_id.to_check) as this will be called quite often.
(We already did this for journal_id at https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_move.py#L638)